<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="framePlayer">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="text">
     <string>frame1</string>
    </property>
   </widget>
   <widget class="QLabel" name="framePlayer2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>220</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="text">
     <string>frame2</string>
    </property>
   </widget>
   <widget class="QLabel" name="PlayerName">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>220</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Player Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="Player2Result">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>570</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Script</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PlayerResult2</string>
    </property>
   </widget>
   <widget class="QLabel" name="GameResult">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>570</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Script</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Result</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="PlayerScore">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>630</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Script</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PlayerScore</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Player2Score">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>630</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Script</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Player2Score</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="TimeLabel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>640</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Script</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>time</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="PlayerResult">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>570</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Script</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PlayerResult</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonExit">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QLabel" name="Player2Name">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>100</y>
      <width>220</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Player Name</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>885</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
